src/java.base/share/classes/sun/security/ssl/CertificateAuthoritiesExtension.java

```diff
@@ -25,16 +25,21 @@
 
 package sun.security.ssl;
 
+import sun.security.ssl.SSLExtension.ExtensionConsumer;
```

Comment:
Please keep the previous import order for consistency with other code in this package. All code in `sun.security.ssl` generally orders the imports as `java`, `javax`, then internal (`sun.`), etc.